src/components/MessageList.vue

```diff
@@ -374,9 +374,15 @@ export default {
         onMessageDblClick(event, message) {
             clearTimeout(this.messageClickTmr);
 
-            let userNick = event.target.getAttribute('data-nick');
-            if (userNick) {
-                this.$state.$emit('input.insertnick', userNick);
+            let dataNick = event.target.getAttribute('data-nick');
+            if (!dataNick) {
+                return;
+            }
+
+            let network = this.buffer.getNetwork();
+            let user = network.userByName(dataNick);
+            if (user && user.nick) {
```

Comment:
This will block it from inserting the nick if the user has already left the channel. It should use `dataNick` if the user object doesn't exist